Http.php

```diff
@@ -36,4 +36,27 @@ public function post( $url = '', $args = array() ) {
 		return $response;
 	}
 
+	/**
+	 * GET request.
+	 *
+	 * @param string $url Base URL for requeset (without params)
+	 * @param array  $args Arguments to add to request
+	 * @return mixed
+	 */
+	public function get( $url = '', $args = array() ) {
+		$query_string = '';
+
+		$curl_handle = curl_init();
```

Comment:
@jessedyck Have you tested this on Windows? I remember facing an issue with the POST request.